File: sortcomparison/sorter.go
package main

import (
	"fmt"
	"sort"
)

type Person struct {
	name string
	age  int
}

func (p Person) String() string {
	return fmt.Sprintf("%s is %d years old", p.name, p.age)
}

func main() {
	people := []Person{
		Person{name: "Jason", age: 28},
		Person{name: "Sabah", age: 27},
	}

	sort.Slice(people, func(i, j int) bool {
		return people[i].age < people[j].age
	})

	fmt.Printf("%v\n", people)
}

File: sortcomparison/sorter.py
#!/usr/local/bin/python

class Person:
    def __init__(self, name, age): 
        self.name = name
        self.age = age
    
    def __repr__(self): 
        return "{} is {} years old".format(self.name, self.age)
    

def main():
    people = [
        Person("Jason", 28),
        Person("Sabah", 27)
    ]

    print(sorted(people, key=lambda person: person.age))

main()